src/components/AmountFilter.jsx

```diff
@@ -35,19 +34,17 @@ export default function AmountFilter(props) {
         />
       </Grid>
 
-      {experiments.expShowSlider && (
-        <Slider
-          defaultValue={[0, 10000]}
-          min={0}
-          max={50000}
-          step={100}
-          onChange={(e) => {
-            onMinChange(e.target.value[0]);
-            onMaxChange(e.target.value[1]);
-          }}
-          sx={{ mt: 1 }}
-        />
-      )}
+      <Slider
+        defaultValue={[0, 50000]}
```

Comment:
There is a few problems with this. In order for it to pick up the min/max values, it needs to be controlled but  `defaultValue` stops working if I set a `value` property. 

I could do something like `value={[min, max || RANGE_MAX]}` but im wondering if this could cause some unexpected behavior if the user doesn't set a max